operations/dynamic-templates/isolation-segment.yml

```diff
@@ -0,0 +1,45 @@
+--- # cleanup-params-passed
+iso_group: (( prune ))
+--- # replace-instance-groups
+type: replace
```

Comment:
Can't mix spruce and gopatch styles, when using double-templating.  This needs to be converted to pure spruce merge style.